src/adapter/src/coord.rs

```diff
@@ -1191,9 +1207,14 @@ impl<S: Append + 'static> Coordinator<S> {
     async fn advance_local_inputs(&mut self, advance_to: mz_repr::Timestamp) {
         self.advance_tables.insert(
             advance_to,
-            self.catalog
-                .entries()
-                .filter_map(|e| if e.is_table() { Some(e.id()) } else { None }),
+            self.catalog.entries().filter_map(|e| {
+                // TODO(andrioni): figure out if there's a way to let this not conflict with updates from storaged
```

Comment:
This comment is probably no longer needed?

Reply:
Indeed!